dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowConstants.java

```diff
@@ -86,9 +88,17 @@ private MlflowConstants() {
 
     public static final String MLFLOW_BUILD_DOCKER = "mlflow models build-docker -m %s -n %s --enable-mlserver";
 
-
     public static final String DOCKER_RREMOVE_CONTAINER = "docker rm -f %s";
 
     public static final String DOCKER_RUN = "docker run --name=%s -p=%s:8080 %s";
 
+    public static final String DOCKER_COMPOSE_RUN = "docker compose up -d";
+
+    public static final String DOCKER_COMPOSE_CANCEL = "docker compose down";
```

Comment:
docker-compose down?

Reply:
I think that should be "docker rm -f container-name"? But when we start the container, the task status will be changed to success, we can not cancel the task running.

Reply:
> I think that should be "docker rm -f container-name"? But when we start the container, the task status will be changed to success, we can not cancel the task running.

You need `docker stop container-name` and then `docker rm container-name`, but `docker compose down` should be used here, it will remove all related resources (although you don't have those for now, but it's a good practice to use `docker compose down` if you use `docker compose up` to start the containers..

And I think @caishunfeng 's meaning is `docker-compose down`, note the dash (`-`) between `docker` and `compose`, `docker compose` is new recently in some newer Docker version so it might fail if users are running old Docker version

Reply:
I've fixed it